src/test-helpers/setupTests.ts

```diff
@@ -1 +1,5 @@
 import "jest-styled-components";
+
+process.env.NEXT_PUBLIC_OAK_ASSETS_HOST = "res.cloudinary.com";
+process.env.NEXT_PUBLIC_OAK_ASSETS_PATH = "mock-cloudinary-cloud/image/upload";
+process.env.NEXT_PUBLIC_CLOUDINARY_CLOUD_NAME = "mock-cloudinary-cloud";
```

Comment:
I was initially going to add a `.env.test` but tbh that is more stuff a dev has to deal with and we'd have to mess with CI too.